<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="memoryView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>390</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAReg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblA">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblX">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineXReg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineYReg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSP">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSPReg">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblST">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ST</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSTReg">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPC">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePCReg">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>$0000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAssemble">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Assemble</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExecute">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStep">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="codeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
